examples/run_snowflake_cortex_test_data.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+"""
+Usage:
+    poetry install
+    poetry run python examples/run_snowflake_cortex_with_github.py
+"""
+
+from __future__ import annotations
+
+import airbyte as ab
+#from airbyte._util.google_secrets import get_gcp_secret_json
+from airbyte.caches import SnowflakeCache
+from airbyte._processors.sql.snowflakecortex import SnowflakeCortexSqlProcessor
+from airbyte.strategies import WriteStrategy
+from airbyte_protocol.models import AirbyteMessage
+from typing import Any, Dict
+from airbyte.secrets.google_gsm import GoogleGSMSecretManager
+
+from airbyte_cdk.models import (
+    AirbyteMessage,
+    AirbyteRecordMessage,
+    AirbyteStateMessage,
+    AirbyteStream,
+    ConfiguredAirbyteCatalog,
+    ConfiguredAirbyteStream,
+    DestinationSyncMode,
+    SyncMode,
+    Type,
+    AirbyteStreamState,
+    StreamDescriptor
+)
+
+
+AIRBYTE_INTERNAL_GCP_PROJECT = "dataline-integration-testing"
+secret_mgr = GoogleGSMSecretManager(
+    project=AIRBYTE_INTERNAL_GCP_PROJECT,
+    credentials_json=ab.get_secret("GCP_GSM_CREDENTIALS"),
+)
+
+secret = secret_mgr.get_secret(
+    secret_name="PYAIRBYTE_SNOWFLAKE_PARTNER_ACCOUNT_CREDS",
+)
+assert secret is not None, "Secret not found."
+secret_config = secret.parse_json()
+
+cache = SnowflakeCache(
+    account=secret_config["account"],
+    username=secret_config["username"],
+    password=secret_config["password"],
+    database=secret_config["database"],
+    warehouse=secret_config["warehouse"],
+    role=secret_config["role"],
+)
+
+# create sample catalog 
+stream_schema = {"type": "object", 
+                 "properties": 
+                    {
+                        "str_col": {"type": "string"}, 
+                        "int_col": {"type": "integer"},
+                        "page_content": {"type": "string"},
+                        "metadata": {"type": "object"},
+                        "embedding": {"type": "vector_array"},
+                        }}
+overwrite_stream = ConfiguredAirbyteStream(
+    stream=AirbyteStream(
+        name="myteststream", json_schema=stream_schema, supported_sync_modes=[SyncMode.incremental, SyncMode.full_refresh]
+    ),
+    primary_key=[["int_col"]],
+    sync_mode=SyncMode.incremental,
+    destination_sync_mode=DestinationSyncMode.overwrite,
+)
+catalog = ConfiguredAirbyteCatalog(streams=[overwrite_stream])
+
+
+# create test messages 
```

Comment:
Here str_col & int_col are original columns. The destination will be expected to add three additional columns to the catalog and to the data: "page_content", "metadata" and "embedding" all of which we get from the vector_db_based chunking & embedding logic.